reactor-test/src/test/java/reactor/test/StepVerifierTests.java

```diff
@@ -666,12 +667,11 @@ public void verifyDurationTimeout() {
 	}
 
 	@Test
-	public void verifyNever() {
+	public void verifyNeverWithExpectTimeout() {
 		Flux<String> flux = Flux.never();
 
 		StepVerifier.create(flux)
-		            .expectSubscription()
-		            .thenCancel()
+		            .expectTimeout(Duration.ofMillis(500))
```

Comment:
use `verifyTimeout` here maybe?

Reply:
in order to cover the method?

Reply:
adding a test below `expectTimeoutSmokeTest` to check the `verifyTimeout` method